extensions/amp-bind/0.1/test/test-amp-state.js

```diff
@@ -64,7 +64,9 @@ describes.realWin('AmpState', {
 
     whenFirstVisiblePromiseResolve();
     return whenFirstVisiblePromise.then(() => {
-      expect(fetchStub).calledWith(/* opt_isInit */ true);
+      expect(fetchStub).calledWithExactly(
+          /* isInit */ true, /* isMutation */ false);
+      expect(updateStub).to.not.have.been.called;
```

Comment:
I've stubbed both `fetchSrcAndUpdateState_` and `updateState_` for simpler testing.

I can see how this can be confusing though. Added a comment to the stub instantiations.